services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationService.kt

```diff
@@ -28,19 +32,25 @@ private val logger = KotlinLogging.logger {}
 const val ALLU_APPLICATION_ERROR_MSG = "Error sending application to Allu"
 
 open class ApplicationService(
-    private val repo: ApplicationRepository,
+    private val applicationRepository: ApplicationRepository,
     private val alluStatusRepository: AlluStatusRepository,
     private val cableReportService: CableReportService,
     private val disclosureLogService: DisclosureLogService,
     private val applicationLoggingService: ApplicationLoggingService,
     private val geometriatDao: GeometriatDao,
+    private val permissionService: PermissionService,
+    private val hankeRepository: HankeRepository,
 ) {
+    @Transactional
```

Comment:
If I understand correctly, this method does only database reads. ->  @Transactional(readOnly = true) should provide some performance benefits.

Reply:
Yes.